client/components/DocsList.js

```diff
@@ -0,0 +1,55 @@
+import React, {Component} from 'react';
+import {List, ListItem} from 'material-ui/List';
+import ContentInbox from 'material-ui/svg-icons/content/inbox';
+import ActionGrade from 'material-ui/svg-icons/action/grade';
+import ContentSend from 'material-ui/svg-icons/content/send';
+import ContentDrafts from 'material-ui/svg-icons/content/drafts';
+import Divider from 'material-ui/Divider';
+import ActionInfo from 'material-ui/svg-icons/action/info';
+import Subheader from 'material-ui/Subheader';
+import db from '../../firebase/initFirebase';
+
+
+class DocsList extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      docsInfo: [],
+    };
+  }
+
+  componentDidMount() {
+    const groupId = this.props.groupId;
+    let gatherDocs = [];
```

Comment:
This would be fine as `const` by the way. All `const` means is you can't change the variable. So it will contain a reference to that created array with `const` no matter what. You can still call methods that modify the object.